reV/bespoke/bespoke.py

```diff
@@ -277,6 +303,64 @@ def _parse_output_req(self):
             logger.error(msg)
             raise KeyError(msg)
 
+    def _parse_prior_run(self):
+        """Parse prior bespoke wind plant optimization run meta data and make
+        sure the SAM system inputs are set accordingly."""
+
+        # {meta_column: sam_sys_input_key}
+        required = {'capacity': 'system_capacity',
```

Comment:
Should we document these required columns in the docstring for `prior_meta` above?